src/test/java/teammates/test/cases/ui/browsertests/InstructorFeedbackEditPageUiTest.java

```diff
@@ -150,9 +150,11 @@ private void testEditSessionAction() throws Exception {
         assertEquals(Const.StatusCodes.BACKDOOR_STATUS_SUCCESS, status);
 
         feedbackEditPage = getFeedbackEditPage();
-        feedbackEditPage.isElementSelected(Const.ParamsNames.FEEDBACK_SESSION_RESULTSVISIBLEBUTTON
-                                           + "_atvisible");
-        //feedbackEditPage.verifyHtmlMainContent("/instructorFeedbackEditPublished.html");
+
+        // Ensuring that the settings did not default back to original values after manual publishing
+        assertFalse(feedbackEditPage.isElementSelected(Const.ParamsNames.FEEDBACK_SESSION_RESULTSVISIBLEBUTTON
+                                                       + "_atvisible"));
+
```

Comment:
Just a question, why do we have to test if the publish immediately button isn't selected though? What does the 'original values' here refer to?